extensions/2.0/Vendor/EXT_instance_features/README.md

```diff
@@ -0,0 +1,76 @@
+# EXT_instance_features
+
+## Contributors
+
+* Peter Gagliardi, Cesium
+* Sean Lilley, Cesium
+* Sam Suhag, Cesium
+* Don McCurdy, Independent
+* Marco Hutter, Cesium
+* Bao Tran, Cesium
+* Samuel Vargas, Cesium
+* Patrick Cozzi, Cesium
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 specification. It depends on the [`EXT_mesh_gpu_instancing`](../EXT_mesh_gpu_instancing) and [`EXT_mesh_features`](../EXT_mesh_features) extensions, to define Feature IDs for GPU instances.
+
+## Overview
+
+In most realtime 3D contexts, performance requirements demand minimizing the number of nodes and meshes in an asset. These requirements compete with interactivity, as applications may wish to merge static objects while still supporting interaction or inspection on those objects. A common performance optimizations is GPU instancing, using the `EXT_mesh_gpu_instancing` extension. But this destroys references to distinct objects, their attributes, and their behaviors.
```

Comment:
I made the sentence less "harsh", just saying

> But this does not allow to uniquely identify the instances that are created during rendering. 

because that's the main point here. (Regardless of what the identification is used for...)